converter.go

```diff
@@ -12,6 +12,8 @@ import (
 	"github.com/getsentry/sentry-go"
 )
 
+const ContextKey = "extra"
```

Comment:
Making it a const means you cannot change it right? (Contrary to the readme)

Reply:
fuuuuck 😜

Reply:
see v1.1.1